app/src/main/java/zapsolutions/zap/fragments/SendBSDFragment.java

```diff
@@ -876,6 +882,39 @@ private void calculateFee() {
         }
     }
 
+    /**
+     * Show progress while calculating fee
+     */
+    private void setCalculatingFee() {
+        if(mOnChain) {
+            // On chain fee calculation is very fast, no need for progress indication
```

Comment:
While testing the fee was calculated almost instantly, so I didn't saw a need to add a progress indicator for onchain fee calculation.